requests_toolbelt/__init__.py

```diff
@@ -16,7 +16,7 @@
 __version__ = '0.3.1'
 __version_info__ = tuple(int(i) for i in __version__.split('.'))
 
-from .adapters import SSLAdapter, SourceAddressAdapter
+from .adapters import SSLAdapter, SourceAddressAdapter, FingerprintAdapter
```

Comment:
We kind of discussed it here: https://github.com/sigmavirus24/requests-toolbelt/issues/54

I think we should start trending towards what we discussed in that issue. I know people will open issues regarding it, but I need to update the documentation either way.